policies/templates/gcp_compute_external_ip_access_v1.yaml

```diff
@@ -79,7 +79,6 @@ spec:
             	# Check if instance is in blacklist/whitelist
             	match_mode := lib.get_default(params, "match_mode", "exact")
             	target_instances := params.instances
-            	trace(sprintf("asset name:%v, target_instances: %v, mode: %v, match_mode: %v", [asset.name, target_instances, params.mode, match_mode]))
```

Comment:
I'm not sure we should remove these trace/debug statements. What's the issue with them? I find them helpful if we're debugging a failing policy.

Either way I think we should put this in a separate PR.

Reply:
I undid this change - I removed the traces from the policies I manage (like firewall etc.) to keep them clean.